resources/smpp/ra/src/main/java/org/restcomm/slee/resource/smpp/SmppSessionsImpl.java

```diff
@@ -77,79 +77,59 @@ public SmppTransaction sendRequestPdu(Esme esme, PduRequest request, long timeou
 			throw new NullPointerException("Underlying SmppSession is Null!");
 		}
 
+		SenderThread sender = esmeSenderThreads.get(esme.getName());
+		if (sender == null) {
+			throw new IllegalStateException("Esme sender not found");
```

Comment:
When session is created SMPP Extentions should call SmppSessionHandlerInterface to create new Handler. In case it will not , this situation will occur. In real world it should not happened , but added it just in case something goes wrong with extentions code